db/migrate/20171213161035_add_indexes_on_images.rb

```diff
@@ -0,0 +1,23 @@
+class AddIndexesOnImages < ActiveRecord::Migration[5.0]
+  def up
+    deduped = Image.group(:name, :compute_resource_id, :operatingsystem_id).pluck('min(id)')
+    Image.where.not(id: deduped).each_with_index do |image, i|
+      say "Found images with duplicate name #{image.name} on Compute Resource #{image.compute_resource}!"
```

Comment:
Output does not show the compute resource for me:

![](https://i.imgur.com/1yIqoHS.png)

Reply:
i'll check this